### ISSUE TITLE
Logger: display keys when message is hash

### ISSUE THREAD

**Issue by davydovanton** [hanami/utils#255]
## Current behaviour
For example we have `/books` action where we log all params:

```ruby
class Index
  def call(params)
    # ...
    logger.info(params)
  end
end
```

Now, if we call `/books?filter=title` we will see something like this:

```
[Hanami] [INFO] [2018-02-28 23:15:23 +0300] title
```

For me it's not a super information logger message. Of course I can change my logger to `logger.info(params: params)`. But in this case I'll see something like this:
```
[Hanami] [INFO] [2018-02-28 23:15:23 +0300] {:filter=>'title'}
```

## What I suggest

Allow to use hash message in logger. For example, in case with action we will take something like this:

```
[Hanami] [INFO] [2018-02-28 23:15:23 +0300] filter=title
```

----------------------

For this we need to change default formatter, I can do it if all agree that it will be more useful.
@hanami/core WDYT?

**Comment by davydovanton** [comment]
After discussion with @jodosha: we're waiting "Coloured output" feature and fix this bug 👏

**Comment by jodosha** [comment]
Fixed by #257
